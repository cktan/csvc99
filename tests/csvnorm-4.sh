# Test Case : CSV with newlines in fields 
../csvnorm in/csvnorm-4.csv 
