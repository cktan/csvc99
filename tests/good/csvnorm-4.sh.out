Jane Doe,New\nYork,25
